docs/reference/commandline/node_activate.md

```diff
@@ -0,0 +1,27 @@
+<!--[metadata]>
++++
+title = "node activate"
+description = "The node activate command description and usage"
+keywords = ["node, activate"]
+[menu.main]
+parent = "smn_cli"
++++
+<![end-metadata]-->
+
+# node activate
+
+    Usage:  docker node activate NODE [NODE...]
+
+    Activate a node in the swarm
+
+Activate a node into the swarm. This command targets a docker engine that is a manager in the swarm cluster.
```

Comment:
Let's explain what it means to activate a node. It means that we're taking a node that was paused or drained, and changing it so it's no longer paused and drained, so it can start accepting task assignments.

> This command targets a docker engine that is a manager in the swarm cluster.

This may be a little unclear - it might sound as if you are activating a manager. We could say "This command must be run on a manager node, but may activate any node in the swarm."